walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,132 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    Collection<String> attendees = request.getAttendees();
```

Comment:
We don't usually use "Collection" for declaration. It's too broad and you lose compiler checking for any potential errors. What does "request.getAttendees()" return?

Reply:
The error I got  that motivated me to use a Collection was: 
incompatible types: java.util.Collection<java.lang.String> cannot be converted to java.util.HashSet<java.lang.String>

request.getAttendees() returns an unmodifiable Collection, so I cannot use another declaration type unless do something like this ArrayList<attendees> newList = new ArrayList<>(request.getAttendees()). I'm not sure whether this would be better.